lms/static/sass/multicourse/_dashboard.scss

```diff
@@ -254,6 +254,26 @@
         .course-container{
           border: 1px solid $border-color-l4;
           border-radius: 3px;
+          .xseries-container{
+            margin: ($baseline*0.4) ($baseline/5) 0 ($baseline*0.4);
```

Comment:
Use @include margin($baseline/2), ($baseline/5), 0, ($baseline/2); for RTL purposes.

Also, where possible please try to use $baseline, $baseline/2, $baseline/4, or $baseline*0.75 before resorting to other multiplied conversions of baseline like *0.4, *1.1, etc.